cluster/src/main/java/com/networknt/cluster/LightCluster.java

```diff
@@ -129,10 +130,19 @@ private URI toUri(URL url) {
     }
 
     static class ClusterNotifyListener implements NotifyListener {
+        private String serviceId;
+
+        ClusterNotifyListener(String serviceId) {
+            this.serviceId = serviceId;
+        }
+
         @Override
         public void notify(URL registryUrl, List<URL> urls) {
-            if(logger.isDebugEnabled()) logger.debug("notify is called in ClusterNotifyListener registryUrl = " + registryUrl + " urls = " + urls);
-            if(urls != null && urls.size() > 0) serviceMap.put(urls.get(0).getPath(), urls);
+            logger.debug("registryUrl is: {}", registryUrl);
```

Comment:
registryUrl is redundant, but to avoid changing the interface at this time, keep it for now.

Reply:
I think it is a good idea.